### PR TITLE
Fix some README links

### PR DESCRIPTION
**Pull request by existentialism**
Fixes some broken links found by tests in https://github.com/babel/babel.github.io/pull/1102

### CODE REVIEW

File: packages/babel-plugin-check-es2015-constants/README.md [hunk 1]
```diff
@@ -56,4 +56,4 @@ require("babel-core").transform("code", {
 
 ## Note
 
-This check will only validate consts. If you need it to compile down to `var` then you must also install and enable [`transform-es2015-block-scoping`](../babel-plugin-transform-es2015-block-scoping).
```
Comment (xtuc) [inline packages/babel-plugin-check-es2015-constants/README.md]: I think it was relative to be able to navigate in the documentation on Github.

File: packages/babel-plugin-transform-es2015-typeof-symbol/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # babel-plugin-transform-es2015-typeof-symbol
 
-> ES6 introduces a new native type called [symbols](http://babeljs.io/docs/learn-es6#symbols). This transformer wraps all `typeof` expressions with a method that replicates native behaviour. (ie. returning "symbol" for symbols)
+> ES6 introduces a new native type called [symbols](https://babeljs.io/learn-es2015/#ecmascript-2015-features-symbols). This transformer wraps all `typeof` expressions with a method that replicates native behaviour. (ie. returning "symbol" for symbols)
```
Comment (xtuc) [inline packages/babel-plugin-transform-es2015-typeof-symbol/README.md]: Does your tests catch the nonexistent anchor?

Comment (existentialism) [inline packages/babel-plugin-transform-es2015-typeof-symbol/README.md]: Ya

File: packages/babel-plugin-check-es2015-constants/README.md (hunk 1 above)
Comment (existentialism) [inline packages/babel-plugin-check-es2015-constants/README.md]: Yeah, we have to choose in cases like this whether it's more weird to:

- Jump to website when viewing on GitHub (or npm, etc)
- Jump to GitHub repo when viewing on website

Comment (existentialism) [inline packages/babel-plugin-check-es2015-constants/README.md]: (Or attempt to do crazy relative path replacement on website :D)

Comment (xtuc) [inline packages/babel-plugin-check-es2015-constants/README.md]: I this case it's the website for sure. It is also broken on npm.

**Pull request merged**